libcentrifugo/engine/engineredis/hash.go

```diff
@@ -0,0 +1,26 @@
+package engineredis
+
+import (
+	"hash/fnv"
+)
+
+// Adapted from from https://github.com/dgryski/go-jump package by Damian Gryski.
```

Comment:
Nitpick but this line added at start of comment makes it fil the test of Go standard docs - it no longer begins with the Function name... Would be picked up by go linter for example